angular/core/components/group_page/join_group_button/join_group_button.coffee

```diff
@@ -1,13 +1,13 @@
 angular.module('loomioApp').directive 'joinGroupButton', ->
-  scope: {group: '='}
+  scope: {group: '=', block: '=?'}
```

Comment:
I've repurposed the join group button to be used at the bottom of threads you can't comment on:

![screen shot 2016-05-10 at 10 20 30 am](https://cloud.githubusercontent.com/assets/750477/15132883/e031179a-1698-11e6-83b8-835118fd1aa5.png)

The only change needed there, styling wise, is to add `btn-block` to the button, which unfortunately can't be done from outside the directive without css hackery.